js/components/ArbeidsrettetOppfolging.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { getHtmlLedetekst, getLedetekst, Utvidbar } from 'digisyfo-npm';
+import Veilederpanel from 'nav-frontend-veilederpanel';
+import { Hovedknapp, Knapp } from 'nav-frontend-knapper';
+import history from '../history';
+import Sidetopp from './Sidetopp';
+
+const ArbeidsrettetOppfolging = () => {
+    return (
+        <React.Fragment>
+            <Sidetopp
+                tittel={getLedetekst('infoside-fo.sidetittel')}
+            />
+            <Rad1 />
```

Comment:
Det hadde vært fint med et mer beskrivende navn for disse radene. Jeg forstår ikke helt hva de er for noe :P